### DISCUSSION TITLE
Documentation and translation

### DISCUSSION THREAD

**Discussion by philiprenich** [fail2ban/fail2ban#3710] [General]
Hello,
I was looking for some answers to some configuration questions I had in the Wiki and noticed a lot of usages of "resp." The sentences this abbreviation was used in did not really make sense and I thought maybe this was a programming / Linux term I was unfamiliar with. A quick little search led me to [this stack exchange Q&A](https://ell.stackexchange.com/questions/6491/what-does-resp-mean-in-these-sentences) where it appears "resp." is a translation from German's "beziehungsweise" or "bzw."

I believe the usage in fail2ban and its documentation is struggling with the same issue of direct or literal translation. It appears the sponsor/maintainer (@sebres ?) is German so that would make sense. (I am in no way disparaging that person! Also tagging them to get their input here.)

Not speaking German, I'm not sure if there are multiple ways that "beziehungsweise" could be translated, but in English translating it to "respectively", which is how I read "resp." does not work in any of the cases I've come across so far in the documentation. That abbreviation is rarely, if ever, used in English writing. 

I'd like to find out what the correct meaning is here and submit a PR with updated language that will be easier understood both by native English speakers, and more importantly, by those who aren't native English or German speakers who will probably struggle to substitute the correct translation themselves. 

Thanks!

ps - again, I am not trying to say this work is bad or the person writing the docs is dumb. Language is hard and English has pretty chaotic rules. Documentation is hard, translation is hard. Just want to help the next person along by making things even clearer :)

**Comment by sebres** [comment]
I mostly use "respectively" as a replacement for "jeweils" and not "bzw" directly (however the first can be surely replaced with second in German. (And I saw often the abbreviation "resp." in English in the same manner as "bzw." will be used in German.)
And I'm trying to use "or else" for "bzw." (but surely cannot guarantee that "resp." wouldn't arise half-automatically)...
But so regarding me in these days. 
No idea how it was in year 2010, 2000 or even earlier.

And welcome with PR... :) 
Unfortunately it works in both directions - hopefully, you'll correctly understand my Denglish (and what I could mean in the documentation/wiki/wherever) and hereafter I'll understand the "fixed" edition properly too. 
Otherwise we will just confuse people even more.

Not to mention, that this (the rephrasing as well as the review) is, to some extent, a unappreciative work...

**Reply by sebres** [reply]
> It appears the sponsor/maintainer (sebres?) is German ...

Yes (as regards the "German").
And no, I am neither a sponsor, nor a maintainer. I am a dev (and co-owner).
